src/go/devcontainer-feature.json

```diff
@@ -15,6 +15,14 @@
             ],
```

Comment:
Can we bump the minor version in here as we are adding a new `option`? (ie. `1.1.0`) 
https://github.com/devcontainers/features/blob/69743e604e5c9ea03eb7d88fef1676b4324549e9/src/go/devcontainer-feature.json#L3
The Go feature will automatically get published once this PR is merged. (Provided the version has changed)

Reply:
Updated